### PR TITLE
Feat/setup

### PR DESCRIPTION
**Pull request by BCerki**

### CODE REVIEW

File: tests/TestingHelper.test.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+describe("TestingHelper", () => {
+  beforeEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  it("reinit function creates an environment, errorContext, and mock router", () => {
+    const testingHelper = new TestingHelper();
+    testingHelper.reinit();
+    expect(testingHelper.environment).toEqual(expect.anything());
```
Comment (BCerki) [inline tests/TestingHelper.test.tsx:19]: I would have liked to test that if was of type `RelayMockEnvironment` but jest didn't like `toBeInstanceOf`

File: tests/TestingHelper.test.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+  it("expectMutationToBeCalled function returns a message when the expected mutation is not called", () => {
+    const testingHelper = new TestingHelper();
+    testingHelper.reinit();
+
+    const environment = createMockEnvironment();
+    commitMutation(environment, {
+      mutation: CreateTodoMutation,
+      variables: {},
+    });
```
Comment (BCerki) [inline tests/TestingHelper.test.tsx:55]: This should fire off the mutation, yes? The testing helper thinks no mutations have been called

Comment (dleard) [reply tests/TestingHelper.test.tsx:55]: you have 2 different environments here.
testingHelper.reinit() creates a mock environment. 
Then line 51: const environment = createMockEnvironment() creates another one.

File: tests/TestingHelper.test.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    );
+  });
+
+  it("expectMutationToBeCalled function returns a message when the expected mutation is not called", () => {
+    const testingHelper = new TestingHelper();
+    testingHelper.reinit();
+
+    const environment = createMockEnvironment();
+    commitMutation(environment, {
+      mutation: CreateTodoMutation,
```
Comment (dleard) [inline tests/TestingHelper.test.tsx:53]:
```suggestion
    commitMutation(testingHelper.environment, {
      mutation: CreateTodoMutation,
```

**Review by tmastrom — APPROVED** [review]

**Pull request merged**